main/webapp/modules/core/scripts/facets/list-facet.js

```diff
@@ -525,6 +526,18 @@ ListFacet.prototype._doEdit = function() {
   new ClusteringDialog(this._config.columnName, this._config.expression);
 };
 
+ListFacet.prototype._editTitle = function() {
+  var currentFacetTitle = this._config.name;
+  
+  var promptText = "The current name of the facet is: "+ currentFacetTitle +".\n What name do you want now?";
```

Comment:
```suggestion
  var promptText = "Current name of facet is: "+ currentFacetTitle +".\n Rename to: ";
```
Let's use Simple English as much as possible for easier translations.  Can we translate this also?